ConditionsConsumed/runCMSDrivers_mc2022.sh

```diff
@@ -0,0 +1,83 @@
+#2: SIM
+echo "------------------------------"
+echo "Running step-1/7: GEN-SIM"
+echo "------------------------------"
+cmsDriver.py TTbar_14TeV_TuneCP5_cfi  -s GEN,SIM -n 5 --conditions auto:phase1_2022_realistic --beamspot Run3RoundOptics25ns13TeVLowSigmaZ --datatier GEN-SIM --eventcontent FEVTDEBUG --fileout output_step2_SIM.root --geometry DB:Extended --era Run3 --relval 9000,100 --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step2_SIM.log
+
+#3: DIGI
+echo "------------------------------"
+echo "Running step-3/7: DIGI "
+echo "------------------------------"
+cmsDriver.py step1  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s DIGI:pdigi_valid --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step2_SIM.root --fileout output_step3_DIGI.root  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' --outputCommands 'keep *_simCastorDigis_*_*' |& tee output_step3_DIGI.log
+
+#4: L1
+echo "------------------------------"
+echo "Running step-2/7: L1"
+echo "------------------------------"
+cmsDriver.py step2  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s L1 --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step3_DIGI.root --fileout output_step4_L1.root --customise_commands="process.load('Configuration.StandardSequences.Digi_cff') \n process.GlobalTag.DumpStat =cms.untracked.bool(True)" --outputCommands "keep *" |& tee output_step4_L1.log
+
+
+
+#4.1: L1+DIGI
+cmsDriver.py step3  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s DIGI:pdigi_valid,L1 --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step2_SIM.root --fileout output_step3_DIGI_L1.root  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' --outputCommands 'keep *_simCastorDigis_*_*' |& tee output_step3_DIGI_l1.log
+
```

Comment:
Which of these two steps will get picked up by the script `getAlCaCondInGT.py` ? I think you should keep only the cmsDriver commands that produce the output file that will be used by the script.